.github/CODEOWNERS

```
@@ -32,6 +32,7 @@
 /pytorch_lightning/plugins              @tchaton @SeanNaren @awaelchli @justusschock
 /pytorch_lightning/profiler             @williamfalcon @tchaton @borda @carmocca
 /pytorch_lightning/profiler/pytorch.py  @nbcsm @guotuofeng
+/pytorch_lightning/strategies           @tchaton @SeanNaren @awaelchli @justusschock
```

We should add @kaushikb11 here too?
```suggestion
/pytorch_lightning/strategies           @tchaton @SeanNaren @awaelchli @justusschock @kaushikb11
```